src/main/java/seedu/address/model/person/ApplicantStatus.java

```diff
@@ -27,13 +25,14 @@ public class ApplicantStatus extends Status {
      */
     public ApplicantStatus(String status) {
         requireNonNull(status);
-        value = matchStatus(status.toLowerCase(), MESSAGE_CONSTRAINTS);
+        checkArgument(isValidStatus(status.toLowerCase()), MESSAGE_CONSTRAINTS);
+        value = status.toLowerCase();
     }
 
     /**
      * Checks and returns the status if it is valid.
      */
-    public String matchStatus(String status, String message) {
+    public static boolean isValidStatus(String status) {
```

Comment:
Nice job changing the method to use boolean instead. Props for remembering the coding standard for naming as well